website/docs/reference/resource-properties/identifier.md

```diff
@@ -55,3 +55,37 @@ Will get compiled to:
 ```sql
 select * from postgres_backend_public_schema.orders
 ```
+
+### Reference sharded tables in BigQuery
```

Comment:
Just for SEO purposes, I can imagine someone googling "sharded table source bigquery"
```suggestion
### Reference sharded tables as a source in BigQuery
```

Reply:
Good call!